roles/ethereum_node/defaults/main.yaml

```diff
@@ -24,6 +24,10 @@ ethereum_node_cl_validator_fee_recipient: "0xF29Ff96aaEa6C9A1fBa851f74737f3c069d
 # Fact discovery
 ethereum_node_fact_discovery_enabled: true
 
+# Mev boost
+ethereum_node_mev_boost_enabled: false
+ethereum_node_mev_boost_http_port: 18550
```

Comment:
I think it's fine to remove this `ethereum_node_mev_boost_http_port` variable . If someone really wants to change the mev boost server port then they can directly use the role specific var `mev_boost_server_port`. If you remove this var, you also need to remove the line where it's referenced in `roles/ethereum_node/tasks/main.yaml`

Reply:
done :D